esphome/components/htu21d/sensor.py

```diff
@@ -35,6 +44,11 @@
                 device_class=DEVICE_CLASS_HUMIDITY,
                 state_class=STATE_CLASS_MEASUREMENT,
             ),
+            cv.Required(CONF_HEATER): sensor.sensor_schema(
```

Comment:
This should not be required. (And I just noticed the temp and humidity are `Required` which I will change in a separate PR.
```suggestion
            cv.Optional(CONF_HEATER): sensor.sensor_schema(
```